### PR TITLE
FSET-1791 Allow Assessors to add availability for wfh

### PR DESCRIPTION
**Pull request by benjstephenson**
Artificially add a day aggregated event for the "Home" location for each day that has another event running.

### CODE REVIEW

**Comment by andrewrembrandt** [comment]
LGTM

File: app/controllers/DayAggregateEventController.scala [hunk 1]
```diff
@@ -47,8 +46,9 @@ trait DayAggregateEventController extends BaseController {
   }
 
   private def find(location: Option[Location], skills: Seq[SkillType] = Nil) = {
-    eventsRepository.getEvents(None, None, location, skills).map {
-      _.groupBy(e => DayAggregateEvent(e.date, e.location)).keys.toList
+    eventsRepository.getEvents(None, None, location, skills).map { events =>
+      events.distinctTransform(_.date, e => DayAggregateEvent(e.date, Location("Home"))) ++
```
Comment (vglushak) [inline app/controllers/DayAggregateEventController.scala]: Not verbose. I don't think distinctTransform function needed. It took me a while to understand what is that.  Basically you can do groupBy date and then map to DAE with Home Location.

Comment (vglushak) [reply app/controllers/DayAggregateEventController.scala]: Do we need to put Home value in yaml file? There are places in code where we validate Venues/locations against  available values there.

Comment (benjstephenson) [reply app/controllers/DayAggregateEventController.scala]: Home is a valid location now, and there are places across the code base where the input location is validated against the yaml file so will throw exceptions if it's not there.

**Comment by vglushak** [comment]
I think you should also change `findAvailableAssessorsForLocationAndDate`, otherwise assessors who share their availability for Home won't comes up on a list for event with Location X.

**Comment by benjstephenson** [comment]
👍  I'd missed these changes you mentioned in the last push.

**Pull request merged**